File: patterns/c-patterns/pattern41.c
// Pattern 41: ...

// #*#*#
// #*#*
// #*#
// #*
// #

#include <stdio.h>
int main()
{
    int i, j;
    int n = 5;

    for (i = n; i >= 1; i--)
    {
        for (j = 1; j <= i; j++)
        {
            if (j % 2 == 0)
                printf("*");
            else
                printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern41.js
// Pattern 41: ...

// #*#*#
// #*#*
// #*#
// #*
// #

const n = 5;

for (let i = n; i >= 1; i--) {
    let row = '';

    for (let j = 1; j <= i; j++) {
        if (j % 2 === 0) {
            row += '*';
        } else {
            row += '#';
        }
    }

    console.log(row);
}
